### ISSUE TITLE
Quick Auto Fill fills nothing on macOS

### ISSUE THREAD

**Issue by baobao1270** [keepassium/KeePassium#259]
**Description**
The quick fill does now work on macOS. The quick fill window shows, but when the TouchID is pressed, nothing is filled.

https://user-images.githubusercontent.com/7949769/193558278-220cca61-2d12-4fe2-a53c-90f82ffe5f63.mov

**How to reproduce**
1. Download KeePassium on MacOS
2. Open any website and setup quick auto fill
3. Reboot the macOS, than repeat step 2

**Expected behavior**
The KeePassium works like the iCloud Password Manager

Below is the screen recording of what iCloud Password Manager works like

https://user-images.githubusercontent.com/7949769/193559530-c2dc11c6-7956-4248-b575-8f6eab4fca9b.mov

**Environment:**
 - Device: MacBook Air M2 2022
 - OS: macOS 12.6
 - App Version: 1.36

**Comment by keepassium** [comment]
Thanks! It looks like AutoFill might be running out of memory ([AutoFill memory limits](https://keepassium.com/articles/autofill-memory-limits/)).

Does it work if you click "KeePassium…" below the suggested account?

**Comment by baobao1270** [comment]
Hi,

I think this might not caused by memory limit. The same database and same version of KeePassium works correctly on my iPad and iPhone. Only the on macOS can reproduce the problem.

The `KeePassium` in the suggested account list works.

I found when I turn the AppLock off, the quick autofill works again.

**Comment by moenodedev** [comment]
I also have to click "KeePassium..." in order to get AutoFill to work. After Touch ID the fields are filled. The database has only a few entries and they contain username, password, email, url, and otp fields.

**Comment by keepassium** [comment]
Thanks for the feedback! Can you please check if the issue persists in the latest 1.37.118? (it should be both in TestFlight and App Store now.)

**Comment by moenodedev** [comment]
Yes it still happens on 1.37.118.

**Comment by cryptosteve2** [comment]
Same here with Keepassium  v1.42.129 Pro
Suggested autofill entry does not do anything, choosing "Keepassium Pro ..." works as expected.

**Comment by keepassium** [comment]
@cryptosteve2 , thanks!

**Comment by meckie-meck** [comment]
I encounter the same problem with KP 1.44.134 on macOS 13.4 on a MacBook Pro 14 2023. But it works on iOS and iPadOS.

**Comment by blinskey** [comment]
This bug is now fixed for me after upgrading to macOS Sonoma 14.0, but there is a new, related problem: a "KeePassium AutoFill" dialog is displayed _after_ the autofill successfully executes. Sorry if this would be more appropriate for a new issue, but I figured I should note it here since it appears to be related to the previously buggy behavior.

Here's the exact sequence of events I'm seeing:

1. When a login field is in focus, the macOS autofill prompt is displayed.
2. I authenticate using TouchID.
3. The login fields are correctly auto-filled and the form is submitted.
4. A TouchID dialog is displayed, with the message "KeePassium AutoFill is trying to Unlock KeePassium." This persists until I authenticate again with TouchID to dismiss the dialog.

I'd be happy to provide video or the details of my configuration if that would be helpful. One thing that might be noteworthy is that I've enabled auto-fill for KeePassium but _not_ iCloud Keychain in the macOS settings. That wasn't possible in macOS 13.

**Comment by keepassium** [comment]
@blinskey, thank you for the feedback! 

It looks like the lifecycle of AutoFill process has changed on Sonoma. There are several reports that indicate the AutoFill keeps running in background, although the system is supposed to terminate it between the invocations. As a result, strange things happen. I will look into this.

**Comment by ezamelczyk** [comment]
@keepassium yeah it seems autofill in general is busted on sonoma. Passwords section in system settings does not pick up any passwords from external providers. I tested keepassium and 1password and both provide nothing.
